packages/plugin-apex/tsconfig.json

```diff
@@ -11,6 +11,7 @@
     "outDir": "./lib",
     "preserveConstEnums": true,
     "strict": true,
+    "skipLibCheck": true,
```

Comment:
Was this change necessary for any particular dependency?

Reply:
Having issue with @oclif/parser

```
$ shx rm -rf lib && tsc -b
node_modules/@salesforce/command/node_modules/@oclif/command/node_modules/@oclif/config/node_modules/@oclif/parser/lib/index.d.ts:16:60 - error TS2344: Type 'TArgs' does not satisfy the constraint 'any[]'.
  Type '{ [name: string]: string; }' is missing the following properties from type 'any[]': length, pop, push, concat, and 26 more.

16 }>(argv: string[], options: Input<TFlags>): Output<TFlags, TArgs>;
 ```

Reply:
@mshanemc or other keepers of salesforce/command, do you have any thoughts on why this might be an issue?